app/assets/javascripts/models/beta/project.js

```diff
@@ -1,9 +1,17 @@
 import * as Label from './label';
+import PropTypes from 'prop-types';
+import {sprintVelocity} from './iteration'
```

Comment:
Add spaces between `{}` 💅 
```suggestion
import { sprintVelocity } from './iteration'
```